.github/workflows/image-push-master.yml

```diff
@@ -29,5 +31,66 @@ jobs:
           push: true
           tags: |
             ${{ env.IMAGE_NAME }}:latest
+            ${{ env.IMAGE_NAME }}:latest-amd64
             ${{ env.IMAGE_NAME }}:${{ github.sha }}
           file: images/Dockerfile
+
+  build-and-push-arm64-device-plugin:
+    if: github.repository_owner == 'k8snetworkplumbingwg'
+    name: Image push ARM64
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push sriov-network-device-plugin
```

Comment:
naive question, what will actually cause the image to be built for ARM64 ?
previously, in travis, you would have asked for an arm64/ppc VM and just build your image there.

With GH actions you get a x86_64 VM and run docker build.
while buildx supports multi-platform build, you need to explicitly request for it right ?

Here is what i found : https://github.com/docker/build-push-action/blob/master/docs/advanced/multi-platform.md

Reply:
@adrianchiris You're correct. It wasn't correct originally. Corrected now.